docs/root/faq/configuration/tracing.rst

```diff
@@ -0,0 +1,10 @@
+.. _common_configuration_tracing:
+
+How do I configure tracing?
```

Comment:
Can you also link to the generic tracing documentation? https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing

Thank you

/wait

Reply:
ok， I will try